src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/ReferenceTokenCompositeSearchParameterV2RowGenerator.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Core.Features.Search.SearchValues;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.TvpRowGeneration
+{
+    internal class ReferenceTokenCompositeSearchParameterV2RowGenerator : CompositeSearchParameterRowGenerator<(ReferenceSearchValue component1, TokenSearchValue component2), ReferenceTokenCompositeSearchParamTableTypeV2Row>
```

Comment:
>ReferenceTokenCompositeSearchParameterV2RowGenerator [](start = 19, length = 52)

They consumed via reflection, and this is reason why `ReferenceTokenCompositeSearchParamTableType_1` got removed from sql script, is this correct assumption?

Reply:
We removed `ReferenceTokenCompositeSearchParamTableType_1` from the "snapshot" 7.sql SQL script because it has been superseded by `ReferenceTokenCompositeSearchParamTableType_2`. 

We codegen a class and a struct from the SQL DDL files for each table type, in this case `ReferenceTokenCompositeSearchParamTableTypeV2TableValuedParameterDefinition` and `ReferenceTokenCompositeSearchParamTableTypeV2Row` respectively.

The row generator in this file is registered as a service in the dependency injection container and consumed by the generated `UpsertResourceTvpGenerator` class.